src/Angor/Client/Pages/Spend.razor

```diff
@@ -183,8 +191,77 @@
                 </div>
             </div>
         </div>
+    
 
     }
+@{
```

Comment:
All this C# code should go in to the `@code {` section bellow, why have two sections? it is just confusing.

Reply:
You are correct, i did it out of comfort to not scroll down to that section but didn't return it.
either way it seems like because it was inside the <div> so everytime you expanded it would calculate it again.
so i moved it to a function called CalculateTotalValues.
good catch